src/scripts/main.js

```diff
@@ -1,3 +1,282 @@
+      newString += value[0].toUpperCase() + value.slice(1).toLowerCase() + ' ';
+    }
+  }
+
+  return newString.trim();
+}
+
+function convertSalary(string) {
+  return `$${string.replace(/\B(?=(\d{3})+(?!\d))/g, ',')}`;
+}
+
+function showNotification(condition, message) {
+  const notification = document.createElement('div');
+  const notificationTitle = document.createElement('h2');
+  const notificationDescription = document.createElement('p');
+
+  if (body.children[3]) {
+    body.children[3].remove();
+  }
+  notification.className = 'notification ' + condition;
+  notification.setAttribute('data-qa', 'notification');
+  notificationTitle.className = 'title';
+  notificationTitle.textContent = convertString(condition);
+  notificationDescription.textContent = message;
+  notification.append(notificationTitle);
+  notification.append(notificationDescription);
+  body.append(notification);
+  setTimeout(() => notification.remove(), 3000);
+}
+
+function nameVerification(value) {
+  if (value.length < 4) {
+    notificationMessage = 'Name must contain more than three letters!';
+    showNotification(error, notificationMessage);
+
+    return true;
+  }
+}
+
+function ageVerification(age) {
+  if (+age < 18) {
+    notificationMessage = 'The employee must be over than 18 years old!';
+    showNotification(error, notificationMessage);
+
+    return true;
+  }
+
+  if (+age > 90) {
+    notificationMessage = 'The employee must be under than 90 years of age!';
+    showNotification(error, notificationMessage);
+
+    return true;
+  }
+}
+
+const body = document.body;
+const thead = document.querySelector('thead');
+const titles = thead.rows[0].children;
+const tbody = document.querySelector('tbody');
+const employes = tbody.rows;
+let clickedTitle = null;
+
+// Сортування списку
+
+thead.addEventListener('click', e => {
+  const title = e.target.closest('th');
+  const titleIndex = [...titles].indexOf(title);
+  const sortedTbody = document.createElement('tbody');
+
+  const sortedEmployes = [...employes].sort((prev, current) => {
+    const prevText = prev.children[titleIndex].textContent;
+    const currentText = current.children[titleIndex].textContent;
+
+    if (convertToNumber(currentText) === 0) {
+      return (clickedTitle === title)
+        ? currentText.localeCompare(prevText)
+        : prevText.localeCompare(currentText);
+    }
+
+    return (clickedTitle === title)
+      ? convertToNumber(currentText) - convertToNumber(prevText)
+      : convertToNumber(prevText) - convertToNumber(currentText);
+  });
+
+  for (const employee of sortedEmployes) {
+    sortedTbody.append(employee);
+  }
+  tbody.innerHTML = sortedTbody.innerHTML;
+
+  clickedTitle = (clickedTitle === title)
+    ? null
+    : title;
+});
+
+// Виділення працівника
+
+tbody.addEventListener('click', e => {
+  const tr = e.target.closest('tr');
+
+  for (const employee of employes) {
+    employee.className = '';
+  }
+
+  tr.className = 'active';
```

Comment:
```suggestion
  tr.classList.toggle('active');
```
Чтобы можно было снять выделение с сотрудника